### PR TITLE
En 1685 fix widget put endpoint

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/widgettype/WidgetService.java [hunk 1]
```diff
@@ -234,6 +242,8 @@ public WidgetDto updateWidget(String widgetCode, WidgetRequest widgetRequest) {
                 }
             }
             this.addFragments(widgetDto);
+        } catch (ValidationGenericException vge) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/widgettype/WidgetService.java]: Log this. No exceptions without logging.

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/widgettype/WidgetService.java (hunk 1 above)
Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/aps/system/services/widgettype/WidgetService.java]: Done!

**Pull request merged**